### PR TITLE
Patch for Adding Epub.js

### PR DESCRIPTION
**Pull request by fchasen**

### CODE REVIEW

File: files/epub.js/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "Fred Chasen"
+github = "https://github.com/futurepress/epub.js"
+homepage = "http://futurepress.org/"
+description = "Epub.js is a JavaScript library for rendering ePub documents in the browser, across many devices."
+mainfile = "build/epub.min.js"
```
Comment (jimaek) [inline files/epub.js/info.ini]: Just `epub.min.js` here

File: files/epub.js/update.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "epub.js",
+  "repo": "futurepress/epub.js",
+  "files": {
+    "include": ["build/epub.min.js", "build/hooks.min.js", "build/reader.min.js", "build/libs/localforage.min.js", "build/libs/zip.min.js"]
```
Comment (jimaek) [inline files/epub.js/update.json]: There is no build directory here https://github.com/futurepress/epub.js/tree/v0.3.2
Only dist/. I guess thats what you need to use

Comment (megawac) [inline files/epub.js/update.json]: local forage is already hosted here http://www.jsdelivr.com/projects/localforage, is your version different?

Comment (fchasen) [inline files/epub.js/update.json]: @jimaek : sorry that release should have been tagged alpha, latest stable is https://github.com/futurepress/epub.js/tree/0.2.9

Comment (fchasen) [inline files/epub.js/update.json]: @megawac : Thanks - I should have checked for those first, will remove.

**Comment by jimaek** [comment]
The problem here now is that the bot will still consider 0.3.1-alpha as the latest version and will fail to add anything

**Comment by fchasen** [comment]
Switched it to `alpha-0.3.1`, will that work?

**Comment by jimaek** [comment]
Not sure honestly. Will merge to see what happens. If it fails we can add 0.2.x manually and when you release 0.3 officially it will pick it up

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**